cmd/skupper/skupper.go

```diff
@@ -440,10 +441,22 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
+				return fmt.Errorf("for the console to work with this user or password, the --console-auth option must be set to internal")
+			}
+
+			if routerCreateOpts.AuthMode == "internal" && !routerCreateOpts.EnableConsole {
```

Comment:
At present I think the restapi option also can use the auth mode. Conceivably you could use it also to secure the collector api even if console was not enabled? I don't think this is necessarily a condition we need to prevent?

Reply:
You are right, should we check that at least --enable-flow-collector is true if the console authentication has any value?

Reply:
Maybe we could warn, but I wouldn't prevent it for now.

Reply:
Updated in 51473eeb635042d0c3aba392f28729c52875aed2